autoagora/model_builder.py

```diff
@@ -11,17 +12,19 @@
 from autoagora.config import args
 from autoagora.indexer_utils import set_cost_model
 from autoagora.logs_db import LogsDB
-from autoagora.utils.constants import AGORA_ENTRY_TEMPLATE
+from autoagora.utils.constants import AGORA_ENTRY_TEMPLATE, MANUAL_AGORA_MODEL_PATH
```

Comment:
Can we add a commandline argument for the model path?

Reply:
sure that definetly sounds better